packages/terra-overlay/README.md

```diff
@@ -4,9 +4,23 @@
 [![NPM version](https://badgen.net/npm/v/terra-overlay)](https://www.npmjs.org/package/terra-overlay)
 [![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.com/cerner/terra-core)
 
-The Terra Overlay component is a component that displays an overlay relative to the container that triggered the overlay. This component blocks interactions and fades out all elements of the triggering container.
+## Deprecation Notice
```

Comment:
if we converted this into mdx, we could use our deprecation notice component: https://engineering.cerner.com/terra-ui/dev_tools/cerner/documentation-components/notice#notices-for-implementation-suggestions

Reply:
We shouldn't convert the readme to mdx. It needs to be renderable by github.

Reply:
and npm

Reply:
oops good call, didn't realize this was the readme